schema/vrs-source.yaml

```diff
@@ -304,34 +331,39 @@ definitions:
   # =============================================================================
 
   # - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - - -
-  # SequenceIntervals
+  # Intervals
 
   SequenceInterval:
     description: >-
-      A SequenceInterval represents a span of sequence. Positions are
-      always represented by contiguous spans using interbase coordinates.
-
-      SequenceInterval is intended to be compatible with that in Sequence Ontology
-      (`SO:0000001<http://www.sequenceontology.org/browser/current_svn/term/SO:0000001>`_),
-      with the exception that the GA4GH VRS SequenceInterval may be zero-width. The SO
-      definition is for an "extent greater than zero".
+      A SequenceInterval represents a span of :ref:`Sequence`. Positions are
```

Comment:
it's not the "span of a Sequence". it's the "span on a Sequence".  We should clarify that the Interval classes are not "sequences" or "segments/spans of sequence" they are merely the interval or span that is used by SequenceLocation and CytobandLocation to specify the start/end of the segment that is being located.

This is the set of classes that we want to roll up into sequenceLocation and cytobandLocation so we can avoid this unnecessary nesting and confusion.

DerivedSequenceExpression is the "span of a sequence"

Reply:
addressed in e2447ba